### DISCUSSION TITLE
[Feature] Auto-Completion for name tagging directly in the People's Tab

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#10288] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It seems like when assigning name to faces directly in the context menu of the people's tab, the completion is not working (not implemented?).
Basically, I would expect the same behavior as when adding a name from a person's page, where the name suggestions get updated the more letters I type.

Without this feature, it leads to typo and duplicate persons, because I missed a capital letter or mispelled something. You only get the chance to merge faces if the spelling is exactly the same, which is not always the case.

Another reason for implementing this feature is that it requires less letters to reach the same name. The user doesn't have to type everything again.

Thanks.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5206
